.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+.idea/
```

Comment:
**.idea/** is the folder that IntelliJ IDEs add for project metadata. In my case, [WebStorm](https://www.jetbrains.com/webstorm/).